### PR TITLE
DOCS-11192 RTF supports one-way TLS only

### PR DESCRIPTION
**Pull request by IsaacEldridge**

### CODE REVIEW

File: modules/ROOT/pages/enable-inbound-traffic.adoc [hunk 1]
```diff
@@ -14,18 +14,20 @@ By default, inbound traffic is disabled to prevent consuming unnecessary resourc
...
+For larger traffic loads, use dedicated nodes. The controller and dedicated nodes must have IP addresses that can be reached (are routable and allowed at applicable firewalls) by the external load balancer. Otherwise, traffic cannot be routed from the load balancer to Runtime Fabric.
 ** Know the domain or domains to use for routing traffic to your applications.
 ** Understand that you need to configure an external load balancer to proxy the TCP protocol. The load balancer does not terminate TLS but does allow Runtime Fabric to do so.
 
 * Be able to configure the DNS settings to associate domains that are signed in the TLS certificate to the IP address or addresses served by the external load balancer.
 
 * You can change only the TLS configuration and policies for Runtime Fabrics that you own. For inherited Runtime Fabric instances, the TLS configuration is read-only.
 
+* Runtime Fabric supports only one-way TLS (client to server) verification.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/enable-inbound-traffic.adoc]: @kmendiratta-crm this is the addition in question

**Review by kmendiratta-crm — DISMISSED** [review]
lgtm

File: modules/ROOT/pages/enable-inbound-traffic.adoc (hunk 1 above)
Comment (sarjeet2013) [reply modules/ROOT/pages/enable-inbound-traffic.adoc]: What is this about?

Comment (IsaacEldridge) [reply modules/ROOT/pages/enable-inbound-traffic.adoc]: @sarjeet2013  this is an addition in support of https://www.mulesoft.org/jira/browse/RTF-2756

Comment (sarjeet2013) [reply modules/ROOT/pages/enable-inbound-traffic.adoc]: Ok, then this is not the right section as this page covers inbound traffic and TLS settings etc. 

We should move the new addition of one-way TLS verification to https://docs.mulesoft.com/runtime-fabric/1.9/runtime-fabric-logs#forward-logs-to-external-services which is applicable for external log forwarding only.

Comment (IsaacEldridge) [reply modules/ROOT/pages/enable-inbound-traffic.adoc]: @sarjeet2013 that's my misunderstanding. What if I add this statement to this new section on limitations for external log forwading (which was created on a PR I just merged) https://github.com/mulesoft/docs-runtime-fabric/blob/v1.9/modules/ROOT/pages/runtime-fabric-logs.adoc#limitations-for-external-log-forwarding

Comment (sarjeet2013) [reply modules/ROOT/pages/enable-inbound-traffic.adoc]: @IsaacEldridge Yes, that would work.

**Review by kmendiratta-crm — DISMISSED** [review]

File: modules/ROOT/pages/enable-inbound-traffic.adoc [hunk 2]
```diff
@@ -14,11 +14,11 @@ By default, inbound traffic is disabled to prevent consuming unnecessary resourc
 
 * Be able to create or configure an external TCP load balancer to manage traffic to applications running in Runtime Fabric on VMs / Bare Metal:
 
-** Gather all IP addresses of controller or dedicated nodes (depending on the configuration you choose during installation) that can be reached by the external TCP load balancer. These IP addresses may not be known until you complete the Runtime Fabric on VMs / Bare Metal installation. 
+** Gather all IP addresses of controller or dedicated nodes (depending on the configuration you choose during installation) that can be reached by the external TCP load balancer. These IP addresses may not be known until you complete the Runtime Fabric on VMs / Bare Metal installation.
```
Comment (dukesphere) [inline modules/ROOT/pages/enable-inbound-traffic.adoc]:
```suggestion
** Gather all IP addresses of controller or dedicated nodes (depending on the configuration you choose during installation) that can be reached by the external TCP load balancer. You might not know these IP addresses until you complete the Runtime Fabric on VMs / Bare Metal installation.
```

Comment (dukesphere) [reply modules/ROOT/pages/enable-inbound-traffic.adoc]: I think we generally go for active voice where feasible

File: modules/ROOT/pages/enable-inbound-traffic.adoc [hunk 3]
```diff
@@ -14,11 +14,11 @@ By default, inbound traffic is disabled to prevent consuming unnecessary resourc
...
 +
-For smaller traffic loads, controller nodes can be shared for cluster administration and inbound traffic TLS termination. 
+For smaller traffic loads, controller nodes can be shared for cluster administration and inbound traffic TLS termination.
```
Comment (dukesphere) [inline modules/ROOT/pages/enable-inbound-traffic.adoc]:
```suggestion
For smaller traffic loads, you can share controller nodes for cluster administration and inbound traffic TLS termination.
```

File: modules/ROOT/pages/enable-inbound-traffic.adoc [hunk 4]
```diff
@@ -250,7 +250,7 @@ These values should not be configured.
 *Source IP header name*: Blank +
 *Enable proxy protocol*: Checked
 . Runtime Fabric on VMs / Bare Metal is behind a non-AWS load balancer. +
- If Runtime Fabric on VMs / Bare Metal is deployed behind another type of load balancer, such as F5 or nginx, the source IP address can be provided in an HTTP Header field. In this case, enter the HTTP header name that contains the source IP header. 
+ If Runtime Fabric on VMs / Bare Metal is deployed behind another type of load balancer, such as F5 or nginx, the source IP address can be provided in an HTTP Header field. In this case, enter the HTTP header name that contains the source IP header.
```
Comment (dukesphere) [inline modules/ROOT/pages/enable-inbound-traffic.adoc]: nginx -> NGINX ? https://docs.nginx.com/nginx/admin-guide/load-balancer/http-load-balancer/

**Review by dukesphere — DISMISSED** [review]
small suggestions, NGINX is prob most impt

File: modules/ROOT/pages/enable-inbound-traffic.adoc (hunk 4 above)
Comment (IsaacEldridge) [inline modules/ROOT/pages/enable-inbound-traffic.adoc]:
```suggestion
 If Runtime Fabric on VMs / Bare Metal is deployed behind another type of load balancer, such as F5 or NGINX, the source IP address can be provided in an HTTP Header field. In this case, enter the HTTP header name that contains the source IP header.
```

**Review by dukesphere — APPROVED** [review]

**Pull request merged**